dbt_subprojects/daily_spellbook/models/polymarket/polygon/_schema.yml

```diff
@@ -0,0 +1,444 @@
+version: 2
+
+models:
+  - name: polymarket_polygon_markets
+    meta:
+      blockchain: polygon
+      sector: prediction_markets
+      project: polymarket
+      contributors: tomfutago
+    config:
+      tags: ['polygon','prediction_markets','polymarket','markets']
+    description: "Polymarket: markets"
+    tests:
```

Comment:
```suggestion
    data_tests:
```

Reply:
do a search/replace for all these instances. this was a change in the new version, where they introduced another test type. it's actually a warning on compile, but in CI, we force warnings to error out so we can try to avoid introducing new ones